vaadin-split-layout-flow-parent/vaadin-split-layout-flow-integration-tests/src/test/java/com/vaadin/flow/component/splitlayout/tests/SplitLayoutIT.java

```diff
@@ -128,10 +128,10 @@ public void assertVariants() {
                 By.id("split-layout-theme-variant"));
         scrollToElement(splitLayout);
         Assert.assertEquals(SplitLayoutVariant.LUMO_SMALL.getVariantName(),
-                splitLayout.getAttribute("theme"));
+                splitLayout.getDomAttribute("theme"));
 
         findElement(By.id("remove-variant-button")).click();
-        Assert.assertNull(splitLayout.getAttribute("theme"));
+        Assert.assertNull(splitLayout.getDomProperty("theme"));
```

Comment:
I think we should use `getDomAttribute()` here for consistency as `theme` is only set as attribute.

Reply:
Thanks, done